syntax = "proto3";

package message;

service MessageSender{
  // A simple RPC.
  //
  // Obtains the MessageResponse at a given position.
 rpc GetMessageResponse(Message) returns (MessageResponse) {}

}

message Message{
 string message = 1;
}

message MessageResponse{
 string message = 1;
}